controllers/feeds/reactionList.js

```diff
@@ -63,3 +56,30 @@ module.exports = async (req, res) => {
     });
   }
 };
+
+function sortCommentsCreatedAtAscending(comments) {
+  return comments.sort(
+    (prev, next) => moment(prev.created_at).unix() - moment(next.created_at).unix()
+  );
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sortCommentsCreatedAtAscending` function sorts the comments in place which can lead to unexpected side effects if the original array is used elsewhere. It would be better to create a new array before sorting to avoid mutating the input array.
```diff
-   return comments.sort(
+   return [...comments].sort(
      (prev, next) => moment(prev.created_at).unix() - moment(next.created_at).unix()
    );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->